istantest.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+NPM=$(which npm)
+ROOTPATH=$(pwd)
+TESTBED='testland'
+TESTBEDPATH=$(echo $ROOTPATH/$TESTBED)
+sudo rm -rf $TESTBED
+echo "Instrumenting.."
+istanbul instrument . -o $TESTBED
```

Comment:
Also, you copying all the source files somewhere else seems to confuse coveralls too.